tests/std/tests/P0645R10_text_formatting_parse_contexts/test.cpp

```diff
@@ -2,11 +2,106 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 #include <assert.h>
+#include <concepts>
 #include <format>
 #include <string_view>
 
-// TODO: fill in tests
+using namespace std;
+using namespace literals;

```

Comment:
This is unnecessary - `using namespace std;` also drags in all of the literals. (This is because it's `inline namespace literals`.)